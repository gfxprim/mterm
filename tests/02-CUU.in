2 5
a\n
a\n
\e[0A
