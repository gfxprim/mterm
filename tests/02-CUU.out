 --
|a |
|a |
|  |
|  |
|  |
 --
size 5x2 cursor 1x0
